aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -239,6 +241,128 @@ Tensor wrapped_scalar_tensor(
 
 } // anonymous namespace
 
+Tensor& isin_out(const Tensor& elements, const Tensor& test_elements, bool assume_unique, bool invert, Tensor& out) {
+  out.resize_as_(elements);
+  if (elements.numel() == 0) {
+    return out;
+  }
+
```

Comment:
Should we add fast path for empty test_elements as well?

Reply:
I only added this one because the `TensorIterator` for the default algo breaks if `elements` is empty. Might be worth doing though?

Reply:
No worries, I think we can leave it.